app/services/users/guest.rb

```diff
@@ -0,0 +1,21 @@
+module Users
+  class Guest
+    GUEST_USER_PERMISSIONS = %w[can_read_movie? can_read_person?].freeze
+
+    def method_missing(name, *args, &block)
+      return true if GUEST_USER_PERMISSIONS.include?(name.to_s)
+
+      return false if name.to_s.start_with?('can')
+
+      super(name, *args, &block)
+    end
+
+    private
+
+    def respond_to_missing?(name, include_private)
+      return true if name.to_s.start_with?('can')
+
+      super(name, include_private)
+    end
```

Comment:
remove from private, please

Reply:
why?

Reply:
I think we can break the inheritence under the hood, but I'm not sure, so don't decide at your own peril and risk